chapters/substitution.adoc

```diff
@@ -2,21 +2,36 @@
 .Definition
 Robots are substituted by the <<Robot Handler, robot handler>> of the respective team. No other team member is allowed to take robots out or put robots in.
 
-. A <<Robot Handler, robot handler>> who's team is marked for robot substitution performs at least one of the following operations in the given order.
-.. Put robots in. The maximum allowed number of robots of the team on the field must not be exceeded.
-.. Take robots out. If a robot that is taken out is closer than 1 meter to the intersection of the halfway line with one of the touch lines, another robot can be put in right away. Otherwise the robot handler has to wait until the next substitution opportunity.
+. The <<Robot Handler, robot handler>> should prefer to use long sleeves and colors that won't interfere with the vision system.
+
+. Robots can always be substituted during game play if the following specifications are respected:
+
+.. Is at least partially inside the <<Field Surface, field margin>>
+.. At a distance from the halfway line that must not exceed 30 centimeters
```

Comment:
Prefer SI units here, so 0.3m.

What was the rational again for 0.3m? We should consider that there might be both robot handlers on the same side, as we gave no limitations as to where they should be. Last year, we asked them to be both on the opposite side from the GCO.

To me, 1m would sound just as fine.

Reply:
With 0.3 m there would be a total of 0.6 m (both sides of the halfway line), I believe that's enough for more than one robot being substituted at the same time, but I'm okay with 1 m too